datasets/moroco/moroco.py

```diff
@@ -0,0 +1,175 @@
+# coding=utf-8
+# Copyright 2021 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MOROCO: The Moldavian and Romanian Dialectal Corpus"""
+
+from __future__ import absolute_import, division, print_function
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@inproceedings{ Butnaru-ACL-2019,
+    author = {Andrei M. Butnaru and Radu Tudor Ionescu},
+    title = "{MOROCO: The Moldavian and Romanian Dialectal Corpus}",
+    booktitle = {Proceedings of ACL},
+    year = {2019},
+    pages={688--698},
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+        The MOROCO (Moldavian and Romanian Dialectal Corpus) dataset contains 33564 samples of text collected from the news domain.
+        The samples belong to one of the following six topics:
+            - culture
+            - finance
+            - politics
+            - science
+            - sports
+            - tech
+"""
+
+_HOMEPAGE = "https://github.com/butnaruandrei/MOROCO"
+
+_LICENSE = "CC BY-SA 4.0 License"
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://raw.githubusercontent.com/MihaelaGaman/MOROCO/huggingface-datasets-addition/MOROCO/preprocessed/all/"
```

Comment:
Why not use the link from the official repo ?
You can have something like
```python
_URL = "https://github.com/butnaruandrei/MOROCO/tree/master/MOROCO/preprocessed/train/"

_TRAIN_SAMPLES_FILE = "train/samples.txt"
_TRAIN_LABELS_FILE = "train/category_labels.txt"

_VAL_SAMPLES_FILE = "validation/samples.txt"
_VAL_LABELS_FILE = "validation/category_labels.txt"

_TEST_SAMPLES_FILE = "test/samples.txt"
_TEST_LABELS_FILE = "test/category_labels.txt"
```

Reply:
I just got a better data format in the official repo, so I'll go ahead and just use that link in an upcoming commit.

Reply:
Ok thanks !